src/BoundaryConditions/fill_halo_regions.jl

```diff
@@ -9,8 +9,8 @@ import Base
 ##### General halo filling functions
 #####
 
-fill_halo_regions!(::Nothing, args...) = nothing
-fill_halo_regions!(::NamedTuple{(), Tuple{}}, args...) = nothing
+fill_halo_regions!(::Nothing, args...; kwargs...) = nothing
+fill_halo_regions!(::NamedTuple{(), Tuple{}}, args...; kwargs...) = nothing
```

Comment:
why do we need `; kwargs...`? I don't wee were `fill_halo_regions!` uses kwargs. (not just here, in general).

Reply:
the `async` option in `update_hydrostatic_free_surface_model.jl`

Reply:
should we pass `async`, or are there other kwargs? (it's easier to understand the code if we are more explicit)

Reply:
there is also `reduced_dimensions`.